Docker instructions (#14)
frinyvonnick/node-html-to-image: README.md

});
```

### Using it on Docker

> You have to follow this instructions (https://github.com/puppeteer/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-in-docker) for Docker (Adde to the end of your Dockerfile, there is specific instructiosn for alpine image also) and send this in puppeteerArgs when you call nodeHtmlToImage:

Reviewer: I think the link is enough, we don't need to tell the user he needs to add it at the end of his Dockerfile 😄 

```suggestion
You have to follow this instructions (https://github.com/puppeteer/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-in-docker) to use `node-html-to-image` in  a Docker container. Then, and send this in puppeteerArgs when you call nodeHtmlToImage:
```

await nodeHtmlToImage({
      puppeteerArgs: {
        headless: true,
        args: [
          "--no-sandbox",
          "--remote-debugging-address=0.0.0.0",

>           "--remote-debugging-port=9222",

Reviewer: These options don't seems to be related to the Docker context but there are options to use Chromium devtools. Am I wrong?
